distributed/scheduler.py

```diff
@@ -796,6 +796,14 @@ def __init__(self, name):
                     self.name
                 ]
             )
+        elif "global-duration" in dask.config.get(
+            "distributed.scheduler.default-task-durations"
+        ):
+            self.duration_average = parse_timedelta(
+                dask.config.get("distributed.scheduler.default-task-durations", {}).get(
+                    "global-duration"
+                )
+            )
```

Comment:
I suspect that this was held over from before?

Reply:
Yes, the default-task-durations dict already exists in the config. I don't know if we should change this only in order to add a possibility for configuring all function durations.

Reply:
Are you running into a situation where you need this?  If not, I recommend that we remove it for now.

Reply:
Yes, unfortunately we have an issue as described in #3516 and don't know all function names beforehand as there is also task fusing happening. That's why we would like to be able for individual clusters to change the general default task duration.

Reply:
Can we keep it in the PR in that case @mrocklin ?

Reply:
I'm having trouble understanding this change. Is `global-duration` an existing key in scheduler.default-task-durations?

This also seems independent of the changes to Adaptive, right?

Reply:
I suppose the interaction is through Adaptive using the expected duration to make scaling decisiosn.

Reply:
Yeah, it is related through Adaptive using the expected durations in the occupancy.
Currently the key `global-duration` does not exist in the default-task-durations. As the keys are the names of the executed functions this also should never collide by accident as `global-duration` is not a valid function name.
This key is supposed to be a wildcard.

Reply:
OK, I'm better understanding the motivation for this change, thanks. I'm still a bit unsure about it since it's such a blunt too for the job. I can imagine several alternatives (users running optimize locally to discover keys, better control over names through optimization, task-prefix control) which might be better / conflict with the changes here.

If we do go forward with this, I'm not sure about the `global-duration` though, since duration is already in the parent key. I'd recommend putting it in a separate key like `default-duration: null`.

Regardless, you might want to consider making these changes in a new pull request so that the uncontroversial changes in Adaptive don't get held up.l

Reply:
Yes, you are right this change should go in its own PR. Thanks!